packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -424,8 +509,20 @@ class FloatingActionButton extends StatelessWidget {
   ///  * [Feedback] for providing platform-specific feedback to certain actions.
   final bool? enableFeedback;
 
+
+  /// The spacing between the icon and the label for an extended
+  /// [FloatingActionButton].
```

Comment:
Need to document the theme dependence here

Reply:
Done